static/main.js

```diff
@@ -278,21 +278,35 @@ function drawPrefectureTable(prefectures, totals) {
     }
     return true
   })
-  
+
   dataTable.innerHTML = dataTable.innerHTML + unspecifiedRow
-  
+
   let totalStr = 'Total'
   if(LANG == 'ja'){
     totalStr = '計'
   }
-  
+
   dataTable.innerHTML = dataTable.innerHTML + "<tr class='totals'><td>" + totalStr + "</td><td>" + totals.confirmed + "</td><td>" + totals.recovered + "</td><td>" + totals.deceased + "</td></tr>"
 }
 
+function unDrawKpis(totals, totalsDiff) {
```

Comment:
I think I'm missing something here, not sure why we can't just overwrite the numbers with the newly loaded numbers, instead of needing an 'undraw' function?

Reply:
I do this to show that the data is reloading, so it shows the loading spinners again and then replaces those, i.e. resets to the original page load state before downloading again

Reply:
Is it simple to leave the page completely intact and just overwrite data / redraw graph / prefectures where necessary? I think it would both simplify the code, and be easier on the user? It seems currently the data disappears for as long as the new network request is being loaded/parsed? (or am I misreading).

Thanks!

Reply:
> Is it simple to leave the page completely intact and just overwrite data / redraw graph / prefectures where necessary?

I use it as a [signifier](https://sites.google.com/site/thedesignofeverydaythings/home/signifiers) of sorts. You can test this by uncommenting the `unDrawKpis` on the `index.html` page, if you have it pulled down locally. I had to put a log inside of the `setInterval` to make sure it was working.

Basically, the user doesn't know what's happening unless some visual indication is given. In fact, they'll probably be inclined to refresh the page anyway unless there's some visual countdown to next refresh or a "Last refresh" time